docusaurus/docs/guides/deployments/10-linux/10-controller/10-deploy.mdx

```diff
@@ -18,85 +18,43 @@ We'll cover the following topics:
-
-```text
-sudo bash ./install.bash openziti-controller
-```
+Or, manually configure the package repo and install `openziti-controller`.
 
 <LinuxPackageRepo />
 
+The `openziti` package provides the `ziti` CLI and is installed as a dependency.
+
 ## Configuration
 
-You must provide or generate a configuration. Configuration consists of a PKI, a config YAML file, and a database.
+You must generate, migrate, or craft a configuration. Configuration consists of a PKI, a config YAML file, and a database.
 
 ### Generate a Configuration
 
-You can generate a configuration by enabling bootstrapping during an interactive install which prompts for the most relevant answers, or by setting these values in the answer file.
-
-1. In **/opt/openziti/etc/controller/service.env** (the answer file for the service)
-    1. Set `ZITI_BOOTSTRAP=true` to opt-in to bootstrapping.
-1. In **/opt/openziti/etc/controller/bootstrap.env** (the answer file for bootstrapping the controller)
-    1. Set `ZITI_CTRL_ADVERTISED_ADDRESS` to the FQDN of the controller.
-    1. Set `ZITI_PWD` to the desired management password for user `admin`. You may delete this after bootstrapping.
-
-#### Generating a Public Key Infrastructure
-
-The controller service will generate a root CA and issue an intermediate CA cert during the first startup.
-
-The controller needs a CA to issue certificates for edge identities during enrollment. This CA is known as the edge signer. It's a good idea to secure the root in a different location and use an intermediate CA for this purpose. This allows you to recover in a scenario where the intermediate was lost without changing the root of trust and invalidating all edge enrollments. Refer to [the article about backing up the controller](/guides/deployments/10-linux/10-controller/60-backup.mdx) for more information.
-
-Disable bootstrapping a PKI by setting `ZITI_BOOTSTRAP_PKI=false` in **/opt/openziti/etc/controller/service.env**.
-
-Check out [the PKI page](/learn/core-concepts/pki.md) for an overview of these concepts.
-
-#### Generating a Configuration File
-
-The controller's configuration is loaded from a file ([reference](/reference/30-configuration/controller.md)). The Linux system service will generate a valid configuration during the first startup unless one already exists.
-
-The filename of the configuration file, relative to the Linux service's working directory (default: **/var/lib/ziti-controller/config.yml**). You may override this in **/etc/systemd/system/ziti-controller.service.d/override.conf**.
+This is the recommended approach if you are installing a new controller.
 
-Disable bootstrapping a configuration by setting `ZITI_BOOTSTRAP_CONFIG=false` in **/opt/openziti/etc/controller/service.env**.
+1. Set the required values in the answer file **/opt/openziti/etc/controller/bootstrap.env**.
+    1. `ZITI_CTRL_ADVERTISED_ADDRESS` - control plane permanent DNS name

```

Comment:
so after install, the user would need to edit this file, that's the idea, right? if that's the case, I'd like this to say "edit the answer file and set the required values".  I'd also like to eliminate the ZITI_CTRL_BIND_ADDRESS from this list since i predict < 10% of people will actually care (proabably much less).

Also the ZITI_CTRL_ADVERTISED_ADDRESS can it be: "the address all participants in the overlay will use to reach the controller"? It cannot be mandated that it's a DNS name

Finally, the easy path should be to just run the script and answer the answers when prompted. i don't see that mentioned in here yet, right? Interactive is by far the easiest for people imo

Reply:
It must be a DNS name until https://github.com/openziti/ziti/issues/1993